samples/Demo/speedometer-provider/src/main.rs

```diff
@@ -150,10 +229,20 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     })
     .await?;
 
+    let topic = "dashboard/value/speed";
```

Comment:
add a //TODO with comment to move this into the configuration file

Reply:
Added